apps/discourse/3.3.2/docker-compose.yml

```diff
@@ -33,7 +33,7 @@
       labels:
         createdBy: "Apps"
     sidekiq:
-      image: bitnami/discourse:3.3.2
+      image: bitnami/discourse:3.3.3
       container_name: ${CONTAINER_NAME}-sidekiq
       restart: always
       networks:
```

Comment:
The provided diff shows an update of two images from `bitnami/discourse:3.3.2` to `bitnami/discourse:3.3.3`. This is a normal operation for updating dependencies in Docker configurations and does not introduce any immediate irregularities or potential issues. However, it might be beneficial to review the changes after deployment to ensure nothing goes awry.

If you have specific questions about deploying these containers with Sidekiq as well, feel free to ask!